test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -1693,6 +1693,17 @@ def test_softshrink(self):
         x = torch.tensor(-0.5).to(dtype=torch.float32)
         self.run_test(model, x)
 
+    @skipIfUnsupportedMinOpsetVersion(9)
+    def test_atan2(self):
+        class Model(torch.nn.Module):
+            def forward(self, x, y):
+                return torch.atan2(x, y)
+
+        model = Model()
+        x = torch.randn(4, 4)
```

Comment:
Since quadrant is related in computation, could you modify the test case to test with inputs with combination from different quadrants? Current `randn` always generate positive inputs in range (0, 1).